tests/.eslintrc.js

```diff
@@ -1,5 +1,13 @@
 module.exports = {
   env: {
     'embertest': true
+  },
+  globals: {
+    'Factory': false,
```

Comment:
can't you just declare `/* globals Factory */` at the top of each file where it's used? That should keep linting from complaining.